tests/dragonfly/server_family_test.py

```diff
@@ -250,3 +257,64 @@ async def test_latency_stats_disabled(async_client: aioredis.Redis):
     for _ in range(100):
         await async_client.set("foo", "bar")
     assert await async_client.info("LATENCYSTATS") == {}
+
+
+async def test_metrics_sanity_check(df_server: DflyInstance):
+
+    def on_container_output(container: DockerContainer, fn: Callable):
+        for entry in container.get_logs():
+            for row in entry.decode("utf-8").split("\n"):
+                fn(row)
+
+    def assert_no_error(entry: str):
+        assert "level=ERROR" not in entry, f"error in prometheus log: {entry}"
+        assert "level=WARN" not in entry, f"warning in prometheus log: {entry}"
+
+    # Piggyback on the first known mounted path if running in CI, the container running the test will start another
+    # container with prometheus. The prometheus container needs the file present on the host to be able to mount it.
+    # Fall back to /tmp so the test can be run on the local machine without using root.
+    parent = next((p for p in ("/var/crash", "/mnt", "/tmp") if os.access(p, os.W_OK)), None)
+
+    # TODO use python-docker api to find a valid mounted volume instead of hardcoded list
```

Comment:
This might actually not be possible, because we would have to figure out the mounts from within the container so the docker api will not be helpful. Hardcoding a list seems more appropriate.